### ISSUE TITLE
Compatibility with pathlib

### ISSUE THREAD

**Issue by celynw** [arsenetar/send2trash#49]
First of all, thank you for this.
As an advocate of `pathlib`, I'd like to be able to run
```python
send2trash(path)
```
where `path` is a `pathlib.Path` object.
From what I gather this isn't implemented, because I get:
```none
TypeError: Must be bytes, not PosixPath
```
but it works fine if I do:
```python
send2trash(str(path))
```
It might be as simple as wrapping the input in `str()`, I haven't checked how other libraries are handling it.

**Comment by arsenetar** [comment]
Should be fixed with changes from 94e1ec007a6db5538b170ff27da9a310a8ef7961

**Comment by arsenetar** [comment]
1.8.0 is now available, closing
